Jenkinsfile

```diff
@@ -0,0 +1,25 @@
+def qpc_version = "0.0.47"
+
+node('f28-os') {
+    stage('Install') {
+        sh "sudo dnf -y install origin-clients nodejs"
+        checkout scm
+        sh "sudo setenforce 0"
+    }
+    stage('Build Setup') {
+        sh "node -v"
+        sh "npm -v"
+        sh "sudo npm install -g n"
+        sh "sudo n lts"
+        sh "node -v"
+        sh "npm -v"
+        sh "sudo npm install yarn -g"
+        sh "yarn --non-interactive"
+    }
+    stage ('Test Client') {
+        sh "yarn test"
+    }
+    stage('Build Client') {
+        sh "yarn build"
+    }
```

Comment:
Should this produce a build artifact?  Like a dist dir?

Reply:
Yup, per the off to the side conversation, activating the artifact will become a thing.

The current reason/state it's in it's only running the Asciidoctor setup to confirm everything is good to go build-wise, no errors. A similar setup is in the Travis config.